src/adapter.js

```diff
@@ -445,12 +516,17 @@ export function parse({minim, source, generateSourceMap}, done) {
         }
 
         // Currently, default responses are not supported in API Description format
-        // TODO: [Annotation] Add warning about not showing the default!
         const relevantResponses = _.chain(methodValue.responses)
           .omit('default')
           .omit(isExtension)
           .value();
 
+        if (methodValue.responses && methodValue.responses.default) {
```

Comment:
Just a note, I am in the habit of this as well, but if you have lodash, you can do `_.get(methodValue, 'responses.default`. This saves you from the situation where `responses` is defined, but not an object.

I actually go an error today because of how we checked for some properties. I think we could resolve that with lodash.

Again, out of scope of this PR, but just mentioning.